src/MessageManager.cpp

```diff
@@ -208,7 +208,7 @@ void MessageManager::Unsubscribe( IMessageSubscriber* pSubscriber, MessageID m )
 void MessageManager::Broadcast( Message &msg ) const
 {
 	// GAMESTATE is created before MESSAGEMAN, and has several BroadcastOnChangePtr members, so they all broadcast when they're initialized.
-	if(this != NULL && m_Logging)
+	if(m_Logging) // 'this' won't be NULL
```

Comment:
@teejusb You are wrong.  "this" will be NULL, exactly as the comment explains.  Or at least invalid to access.  Go look at the order that GAMESTATE and MESSAGEMAN are created in and look at what happens when GAMESTATE is created.